packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md

```diff
@@ -4,23 +4,24 @@ title: Basic sign-in flow using the password factor
 
 <ApiLifecycle access="ie" />
 
-> **Note:** In proxy model architectures, where a server-side application using the Embedded SDK is used as a proxy between client applications and Okta servers, a request context for the client applications is required. Security enforcement is expected to be based on the client request context’s IP address and user agent. However, since these values are currently being derived from the server application rather than the client, this enforcement is not available. As a result, network zones or behaviors that drive their conditions based on these request context values (geolocation, IP Address, or user agent) will not work until we can find a solution to the issue.
+> **Caution:** A request context for the client applications is required in proxy-model architectures, where a server-side application uses the Embedded SDK as a proxy between client applications and Okta servers. In this scenario, Okta expects security enforcement to be based on the client request context’s IP address and user agent. Currently these values are being derived from the server application rather than the client, where this security enforcement is not available. As a result, network zones or behaviors that drive their conditions based on these request context values (geolocation, IP Address, or user agent) do not work.
 
-This guide covers a basic user sign-in request, which is the simplest of all use cases and is the first use case that you should try after you install the SDK. The flow diagram and steps describe how to build a simple sign-in form and how to authenticate the credentials.
+This guide explains how to authenticate credentials and handle the response. This is the simplest use cases and the first use case you should try after installing the SDK.
 
 <StackSnippet snippet="pwdoptionalusecase" inline />
 
 ---
 
-**Learning outcomes**
+**What you'll learn**
 
-Understand how to implement basic sign-in using Okta Identity Engine.
+Learn how to authenticate credentials and handle the responses using Okta Identity Engine.
 
 **What you need**
 
 * An app that uses the embedded Okta Identity Engine SDK
+* A sign in page with a form that captures both the username and password
```

Comment:
Consider moving step 1 to the requirements because the guide doesn't explain how to create the form.